doc/webdevicons.txt

```diff
@@ -623,6 +623,18 @@ Extra Configuration ~
 >
   " Adding the custom source to denite
   let g:webdevicons_enable_denite = 1
+
```

Comment:
Thank you for adding these!

However there is a slight wrinkle that is not probably clear. We generate the vim doc in a "fun" way :laughing: 

It comes from the readme + wiki pages! 

So you'd want to add these same things here: https://github.com/ryanoasis/vim-devicons/wiki/Extra-Configuration in the same spot as you have them..

I can also do this. Just making you aware

For how the vim doc is generated: https://github.com/ryanoasis/vim-devicons/wiki/Development-Workflow#new-release-workflow
see the code block under "regenerate the vim doc automatically with html2vimdoc"

I need to work on the wiki a bit I think...

**Summary:** Try to make this change, if not I can do it. Either way. It is just a script that will blow away any ad hoc changes to the vimdoc though :)

Reply:
I didn't know that, I will fix it as soon as possible.

Reply:
Added in wiki: [wiki change](https://github.com/ryanoasis/vim-devicons/wiki/Extra-Configuration/_compare/dde391302d292f228454510d9f6609f26e871a70...a11ffd787e70deba49a6bf760125994972bdbaec)